rtil/src/native/level_state.rs

```diff
@@ -49,6 +49,31 @@ impl LevelState {
             (*LevelState::get_ptr()).level = level;
         }
     }
+
+    pub fn set_start_secs(startsec: i32) {
+        unsafe {
+            (*LevelState::get_ptr()).start_seconds = startsec;
+        }
+    }
+
+    pub fn set_start_partial(sp: f32) {
+        unsafe {
+            (*LevelState::get_ptr()).start_partial_seconds = sp;
+        }
+    }
+
+    pub fn set_end_secs(es: i32) {
+        unsafe {
+            (*LevelState::get_ptr()).end_seconds = es;
+        }
+    }
+
+    pub fn set_end_partial(eps: f32) {
+        unsafe {
+            (*LevelState::get_ptr()).end_partial_seconds = eps;
+        }
+    }
+
```

Comment:
Does every letter in a variable or function name cost you money? ;) Just name everything the same, even if its 5 characters longer. That way I don't need to remember which layer of abstraction had which different name for literally the same function or variable.